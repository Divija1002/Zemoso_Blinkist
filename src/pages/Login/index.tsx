import { Typography, Box } from "@mui/material";
import React, { Fragment } from "react";
import LoginButton from "../../components/authentication/LoginButton";

const Login = () => {
  return (
    <Fragment>
      <Typography variant="h1" sx={{ textAlign: "center" }}>
        You are Logged Out
      </Typography>
      <Box sx={{ paddingX: "800px", paddingTop: "10px" }}>
        <LoginButton />
      </Box>
    </Fragment>
  );
};

export default Login;
