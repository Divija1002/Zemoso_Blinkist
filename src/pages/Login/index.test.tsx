import renderer from "react-test-renderer";
import Login from ".";

describe("Login page Snapshot", () => {
  it("Login page testing", () => {
    const renderedComponent = renderer.create(<Login />);
    expect(renderedComponent).toMatchSnapshot();
  });
});
