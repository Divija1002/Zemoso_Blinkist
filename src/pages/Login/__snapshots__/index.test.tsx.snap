// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Login page Snapshot Login page testing 1`] = `
Array [
  <h1
    className="MuiTypography-root MuiTypography-h1 css-sqoje6-MuiTypography-root"
  >
    You are Logged Out
  </h1>,
  <div
    className="MuiBox-root css-xcbaci"
  >
    <button
      onClick={[Function]}
    >
      Log In
    </button>
  </div>,
]
`;
